modules/backend/classes/NavigationManager.php

```diff
@@ -118,7 +119,7 @@ protected function loadItems()
         /*
          * Sort menu items
          */
-        uasort($this->items, function ($a, $b) {
+        uasort($this->items, static function ($a, $b) {
```

Comment:
@Klaasie is there a particular reason to change this to a static function?

Reply:
@bennothommo The function in this case doesn't require anything from the object scope (`$this`). So declaring this as a static method is a small optimization. (Actually, it's considered a micro optimization)

Reply:
Thanks for the info - I did not know that, but it makes sense.